### PR TITLE
Docs Fixes

### PR DESCRIPTION
**Pull request by TejasQ**
This PR fixes the docs in the following ways:

- Heavily reorganizes the functions docs to represent action functions and custom functions
- Prioritizes generated API client over endpoints
- Makes `Identity` interplay with application users clearer
- Adds many more permissions examples

### CODE REVIEW

File: pages/jobs.mdx [hunk 1]
```diff
@@ -82,6 +82,8 @@ The table below shows examples of the human-readable form with the equivalent cr
 | `"every monday at 9am"`                              | `"0 9 * * 1"`          |
 | `"every tuesday and thursday at 8am, 12pm, and 5pm"` | `"0 8,12,17 * * 2,4"`  |
 
+If you're not familiar with cron syntax you can use a tool like [crontab.guru](https://crontab.guru/) to help you build your schedule.
```
Comment (tomfrew) [inline pages/jobs.mdx:85]: We now support human friendly cron, we should probably add that here at some point - https://github.com/teamkeel/keel/pull/1286

File: pages/permissions.mdx [hunk 2]
```diff
@@ -28,21 +28,83 @@ model MyModel {
...
 Some key things to understand are:
 
 - There can be **many** permission rules in a model or action
 - Action-level rules **override** model-level rules
-- If **any** permission rule passes the action is allowed
-
-### Expressions
+- If **any** permission rule passes, the action is allowed
 
-If provided the `expression` argument is evaluated for every record being acted upon and it has the following in-scope identifiers:
```
Comment (tomfrew) [inline pages/permissions.mdx:45]: I think this is quite important context to explain. That we run that expression against every returned row of data. (although agree the phrasing of that isn't great!)

**Review by tomfrew — APPROVED** [review]
Looks great! 

Left a couple of small comments. Can we also update all heading to sentence case?

File: pages/jobs.mdx (hunk 1 above)
Comment (TejasQ) [reply pages/jobs.mdx:85]: That's already mentioned here: https://github.com/teamkeel/docs/pull/83/files#diff-53ffca5f780a5dc44f7ae9758552040e2c98c750b9b4a4b083acd32292d3ca1bR44

Comment (TejasQ) [reply pages/jobs.mdx:85]: That's mentioned [here](https://github.com/teamkeel/docs/pull/83/files#diff-53ffca5f780a5dc44f7ae9758552040e2c98c750b9b4a4b083acd32292d3ca1bR44).

**Pull request merged**